cloud-install.sh

```diff
@@ -0,0 +1,86 @@
+#!/bin/bash
+
+print_and_log() {
+	echo "$1" | sudo tee /var/log/reNgine.log
+}
+print_and_log "Installing reNgine and it's dependencies"
+
+print_and_log " "
+if [ "$EUID" -ne 0 ]; then
+	print_and_log "Error installing reNgine, Please run this script as root!" | tee -a /var/log/reNgine.log
+	print_and_log "Example: sudo ./cloud-install.sh"
+	exit
+fi
+
+print_and_log " "
+print_and_log "#########################################################################"
+print_and_log "Installing curl..."
+print_and_log "#########################################################################"
+if [ -x "$(command -v curl)" ]; then
+	print_and_log "CURL already installed, skipping."
+else
+	sudo apt update && sudo apt install curl -y
+	print_and_log "CURL installed!!!"
+fi
+
+print_and_log " "
+print_and_log "#########################################################################"
+print_and_log "Installing Docker..."
+print_and_log "#########################################################################"
+if [ -x "$(command -v docker)" ]; then
+	print_and_log "Docker already installed, skipping."
+else
+	curl -fsSL https://get.docker.com -o get-docker.sh && sh get-docker.sh
+	print_and_log "Docker installed!!!"
+fi
+
+print_and_log " "
+print_and_log "#########################################################################"
+print_and_log "Installing docker-compose"
+print_and_log "#########################################################################"
+if [ -x "$(command -v docker-compose)" ]; then
+	print_and_log "docker-compose already installed, skipping."
+else
+	curl -L "https://github.com/docker/compose/releases/download/v2.23.0/docker-compose-$(uname -s)-$(uname -m)" -o /usr/local/bin/docker-compose
```

Comment:
So it probably is fine, although bit weird we install docker in user-data script and then compose here.

Reply:
So the readme of rengine actually recommends installing it before hand. I'm doing it mainly out of "precaution" (ie i'd rather do it "on my own on my end" than purely relying on the script). If at some point this is removed from the install script we'll still be running the core (docker) for rengine